Userland/Libraries/LibArchive/TarStream.h

```diff
@@ -59,9 +59,9 @@ class TarInputStream {
 class TarOutputStream {
 public:
     TarOutputStream(Core::Stream::Handle<Core::Stream::Stream>);
-    ErrorOr<void> add_file(DeprecatedString const& path, mode_t, ReadonlyBytes);
-    ErrorOr<void> add_link(DeprecatedString const& path, mode_t, StringView);
-    ErrorOr<void> add_directory(DeprecatedString const& path, mode_t);
+    ErrorOr<void> add_file(String const& path, mode_t, ReadonlyBytes);
+    ErrorOr<void> add_link(String const& path, mode_t, StringView);
+    ErrorOr<void> add_directory(String const& path, mode_t);
```

Comment:
I haven't looked at the actual implementations of those functions closely, but we might be able to make those a `StringView` right away?